src/FieldRegistry.C

```diff
@@ -54,21 +57,31 @@ Registry()
     {"dwdx",                      SingleStateNodalVector},
     {"edge_area_vector",          SingleStateEdgeVector},
     {"effective_viscosity" ,      SingleStateNodalScalar},
+    {"elemCentroid",              SingleStateElemVector},                          
     {"element_volume",            SingleStateElemScalar},
+    {"elemScalarField",           SingleStateElemScalar},  // Used in testing
```

Comment:
Not for this PR, but a potential refactor that comes to mind is a separate registry for unit tests.
In the mean time, to make this more organized could we stick these unit-test only fields at the end of the registry and do a block comment to indicated fields below are only for tests?

Reply:
Yes we could put all of the testing field names at the end of the std::map constructor.  The alphabetical listing is just a convention.  It does seem like we have quite a few field names and there are probably more to  add. I still think the amount of memory used in these maps is trivial and the map search still quite fast at this size.